graylog2-server/src/main/java/org/graylog2/rest/resources/system/contentpacks/CatalogResource.java

```diff
@@ -78,4 +86,14 @@ public CatalogResolveResponse resolveEntities(
 
         return CatalogResolveResponse.create(entities);
     }
+
+    @POST
+    @Timed
+    @ApiOperation(value = "Get titles of provided entities")
+    @NoAuditEvent("This endpoint does not change any data")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/entity_titles")
```

Comment:
Let's make the naming a bit more RESTful. :slightly_smiling_face: 
```suggestion
    @Path("/entities/titles")
```